docs/tutorials/getting_started.md

```diff
@@ -0,0 +1,135 @@
+future, you might also need additional packages based on your project
+requirements.
+
+```bash
+pip install frequenz-sdk  # Install the Frequenz SDK
+```
+
+## Create a project
+
+### Create a Python file
+
+You can start by simply creating a Python script (e.g., `pv_optimization.py`)
+using your favorite text editor.
+
+### Use Frequenz repo config
+
+As an alternative and for larger projects, it's recommended to set up the
+project using the [Frequenz repo
+configuration](https://frequenz-floss.github.io/frequenz-repo-config-python/latest).
+
+## Import necessary modules
+
+You can now open the app's main file and start adding content. Begin by
+importing the necessary libraries.
+
+```python
+import asyncio
+import os
+
+from datetime import timedelta
+from frequenz.sdk import microgrid
+from frequenz.sdk.actor import ResamplerConfig
+```
+
+## Create the application skeleton
+
+The main logic of your application will run within an async function. Let's
+create a skeleton that contains all the necessary code to initialize a
+microgrid.
+
+```python
+async def run() -> None:
+    # Configuration from environment variables
+    microgrid_host = os.getenv(
+        "MICROGRID_HOST", default="microgrid.sandbox.api.frequenz.io"
+    )
+    microgrid_port = int(os.getenv("MICROGRID_PORT", default="62060"))
+
+    # Initialize the microgrid
+    await microgrid.initialize(
+        microgrid_host,
+        microgrid_port,
+        ResamplerConfig(resampling_period=timedelta(seconds=1)),
+    )
+
+    # Define your application logic here
+    # ...
+```
+
+## Define the `main()` function
+
+Create a `main()` function that will set up and run the `run()` function using
+asyncio.
+
+```python
+def main() -> None:
+    asyncio.run(run())
+
+if __name__ == "__main__":
+    main()
+```
+
+## Implement the application logic
+
+Inside the `run()` function, implement the core logic of your application. This
+will include creating receivers for data streams, processing the data, making
+decisions, and eventually sending control messages to the microgrid system. We
+will cover more details in the following tutorials. For now, let's simply read
+the power measurements from the microgrid's grid meter and print them on the
+screen. The grid meter is a meter that is directly connected to the grid
+connection point.
+
+```python
+async def run() -> None:
+    # Configuration from environment variables
+    ...
+
+    # Define your application logic here
+    grid_meter = microgrid.logical_meter().grid_power.new_receiver()
+
+    async for power in grid_meter:
+        print(power.value)
+```
+
+## Run your application
+
+You're now ready to run your application. When working on an existing
+microgrid, make sure to set the necessary environment variables, i.e.,
+`MICROGRID_HOST` and `MICROGRID_PORT`, before running the script.
+
+```bash
+# Example usage with environment variables
+export MICROGRID_HOST="your_microgrid_host"
+export MICROGRID_PORT=62060
```

Comment:
Maybe use the public sandbox connection details to make it easier to actually run the program easily?